.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,9 @@
+name: "CodeQL Analysis File Ignoring Config"
```

Comment:
How about just "CodeQL Security Analysis" since this file doesn't ignore the config. Also, this is the name of the workflow, not a description of the file

Reply:
Done

Reply:
@U8NWXD PTAL